gtk/gtk.go

```diff
@@ -9506,6 +9516,254 @@ func (v *TreeRowReference) Valid() bool {
 	return gobool(c)
 }
 
+/*
+ * GtkTreeSortable
+ */
+
+// TreeSortable is a representation of GTK's GtkTreeSortable
+type TreeSortable struct {
+	*glib.Object
+}
+
+// ITreeSortable is an interface type implemented by all structs
+// embedding a TreeSortable.  It is meant to be used as an argument type
+// for wrapper functions that wrap around a C GTK function taking a
+// GtkTreeSortable.
+type ITreeSortable interface {
+	toTreeSortable() *C.GtkTreeSortable
+}
+
+// native returns a pointer to the underlying GtkTreeSortable
+func (v *TreeSortable) native() *C.GtkTreeSortable {
+	if v == nil || v.GObject == nil {
+		return nil
+	}
+	p := unsafe.Pointer(v.GObject)
+	return C.toGtkTreeSortable(p)
+}
+
+func (v *TreeSortable) toTreeSortable() *C.GtkTreeSortable {
+	if v == nil {
+		return nil
+	}
+	return v.native()
+}
+
+func marshalTreeSortable(p uintptr) (interface{}, error) {
+	c := C.g_value_get_object((*C.GValue)(unsafe.Pointer(p)))
+	obj := glib.Take(unsafe.Pointer(c))
+	return wrapTreeSortable(obj), nil
+}
+
+func wrapTreeSortable(obj *glib.Object) *TreeSortable {
+	return &TreeSortable{obj}
+}
+
+// TreeIterCompareFunc defines the function prototype for the sort function (f arg) for
+// (* TreeSortable).SetSortFunc
+type TreeIterCompareFunc func(model *TreeModel, a, b *TreeIter, userData interface{}) int
+
+// GetSortColumnId() is a wrapper around gtk_tree_sortable_get_sort_column_id().
+func (v *TreeSortable) GetSortColumnId() (int, SortType, bool) {
+	sort := C.toGtkTreeSortable(unsafe.Pointer(v.native()))
```

Comment:
you call "native" which returns a *C.GtkTreeSortable and then you convert again...

Reply:
see #514